app/src/main/java/org/inspire/breath/data/Patient.java

```diff
@@ -54,7 +56,7 @@ public void setLastName(String lastName) {
         this.lastName = lastName;
     }
 
-    public void setAge(int age) { this.age = age; }
+    public void setBirthDay(String birthDay) { this.birthDay = birthDay; }
```

Comment:
we might not always know the patient birthday (the MSF doc has a test which has the patient touch their ear over their head to determine age) so maybe just a way to set rough age in years?